share/picongpu/tests/XrayScattering/lib/python/picongpu/checks.py

```diff
@@ -0,0 +1,81 @@
+from os.path import join
+import numpy as np
+import openpmd_api as api
+from picongpu.plugins.data import XrayScatteringData
+from is_close import is_close
+
+
+def compare_with_fft(species, bound_electrons, rotation=None):
+    simulation_path = '../../../../'
+
+    # Load pluginOutput
+    xray_scattering_data = XrayScatteringData(simulation_path, species, 'h5')
+    amplitude = xray_scattering_data.get(iteration=0)
+    del xray_scattering_data
+
+    # Load density
+    internal_path = 'simOutput/h5'
+    file_name = 'simData_%T.h5'
+    path_output = join(simulation_path, internal_path, file_name)
+    series_output = api.Series(path_output, api.Access_Type.read_only)
+    i = series_output.iterations[0]
+    e_mesh = i.meshes['e_density']
```

Comment:
@pordyna and @HighIander Is it correct, that the SAXS scattering scales with number of particles and not with its charge? Or is here a scalar factor missing?